src/scripting/LuaSandbox.cpp

```diff
@@ -0,0 +1,455 @@
+#include <stdafx.h>
+
+#include "LuaSandbox.h"
+
+#include <Utils.h>
+
+LuaSandbox::LuaSandbox(sol::state_view aStateView)
+    : m_lua(aStateView)
+{
+}
+
+void LuaSandbox::Initialize(sol::state_view aStateView)
+{
+    // initialize state + environment first
+    m_lua = aStateView;
+    m_env = { aStateView, sol::create };
+
+    // copy whitelisted things from global table
+    constexpr std::array<std::string_view, 55> whitelistedGlobals =
```

Comment:
I think this is a bug waiting to happen, I think it would be best to use a standard C array, std::size will give us the number of elements and we don't have to give it a size manually.

Reply:
std::size is actually not needed as it looks like static arrays are supported by ranged-loops. Didnt know, wanted to use those (so there is not too many things to change when adding something to whitelist) which is the reason I went with std::arrays before. 
I guess this is truly better though, one less thing to think about and this now fully achieves what I wanted to do with this :P